compiler/runtime/OMRRelocationTarget.cpp

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ * Copyright (c) 2020, 2020 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+
+#include "env/FrontEnd.hpp"
+#include "control/Options.hpp"
+#include "control/Options_inlines.hpp"
+#include "env/jittypes.h"
+#include "runtime/TRRelocationRecord.hpp"
+#include "runtime/TRRelocationRuntime.hpp"
+#include "runtime/TRRelocationTarget.hpp"
+
+bool OMR::RelocationTarget::isOrderedPairRelocation(TR::RelocationRecord *reloRecord, TR::RelocationTarget *reloTarget)
+   {
+   switch (reloRecord->type(reloTarget))
+      {
```

Comment:
Thank you for the comment. This method is a dummy method in OMR. It is promoted for backward compatibility, since the relocation records in OpenJ9  have a more thought through  use for it. In particular, applyRelocationAtAllOffsets is promoted without change, so we've just used it in our runtime as a fallthrough. Could you please suggest a desired action, I could axe the method and all logic related to it, reducing the backward compatibility with OpenJ9 or would just a comment along the lines "use if needed" suffice?